dali/pipeline/operator/builtin/external_source.h

```diff
@@ -39,13 +39,21 @@ struct CudaEventWrapper {
       std::terminate();
     }
   }
+
   ~CudaEventWrapper() {
     auto res = cudaEventDestroy(event);
     if (res != cudaSuccess) {
       DALI_ERROR("Fatal error:: " + to_string(res) + " in ~CudaEventWrapper()");
       std::terminate();
     }
   }
+
+  DEFAULT_COPY_MOVE_ASSIGN(CudaEventWrapper);
```

Comment:
Remove this entire class.
Use `CUDAEvent` from dali/core instead.

Reply:
It is a legacy code, created before CUDAEvent was avaialble.